lib/ansible/cli/galaxy.py

```diff
@@ -82,7 +82,7 @@
 SERVER_ADDITIONAL = {
     'api_version': {'default': None, 'choices': [2, 3]},
     'validate_certs': {'cli': [{'name': 'validate_certs'}]},
-    'timeout': {'default': '60', 'cli': [{'name': 'timeout'}]},
+    'timeout': {'default': C.GALAXY_SERVER_TIMEOUT, 'cli': [{'name': 'timeout'}]},
```

Comment:
wouldn't it be simpler to just default here to the cliargs timeout value?

Reply:
I'm not sure what this would simplify. The cli aargs haven't been parsed yet and since timeout is in the cli config def here, it's pulled in and takes precedence over the ini and env sources.

Reply:
I did wonder if I should just make the default None here though, and handle the default completely in `run` (because the dynamic default doesn't play nicely with unit tests) but since this didn't require any manual patching in `run` for configured servers, I thought it was easier to follow.

Reply:
it does seem like doing x2 work as you set the default here and for cliargs